### DISCUSSION TITLE
Multi-depot VRPs not enabled?

### DISCUSSION THREAD

**Discussion by deniz7z** [PyVRP/PyVRP#689] [Q&A] [answered]
First of all, thanks for this amazing package! Does PyVRP support multi-depot VRPs? 

If I attempt to implement the multi-depot VRP from the [examples](https://pyvrp.org/examples/quick_tutorial.html), I get the following error.

![image](https://github.com/user-attachments/assets/dfa145c1-601f-49eb-adc6-b7766ddaaa9f)

**Comment by leonlan** [comment]
Hi @deniz7z! Multi-depot was added in PyVRP v0.7.0, so I assume you are using an older version of PyVRP. You can check your version of PyVRP [like this](https://pyvrp.org/setup/getting_help.html#submitting-a-bug-report): 

```python
import pyvrp
pyvrp.show_versions()

```

**Reply by deniz7z** [reply]
Thanks for your quick response, @leonlan! 

I indeed am running an older version 0.6.3. Interesting, because I downloaded the package a couple of weeks ago. Is there any pip command to update the package? the following command from the website seems to raise an error:

(base) C:\Users\20182220> pip install 'pyvrp @ git+https://github.com/PyVRP/PyVRP'
ERROR: Invalid requirement: "'pyvrp"

**Reply by leonlan** [reply]
You can try this:

```
pip install --upgrade pyvrp
```

**Reply by N-Wouda** [reply, marked as answer]
@deniz7z are you running Python 3.8? If so, 0.6.x was the last version of PyVRP supporting that Python version. Newer releases of PyVRP target 3.9 and up (3.10 since PyVRP 0.10.0).

**Reply by deniz7z** [reply]
That seems to refer to the same version. What is the newest version? Maybe i could download a specific version.

`(base) C:\Users\20182220>pip install --upgrade pyvrp`
`Requirement already satisfied: pyvrp in c:\users\20182220\anaconda3\lib\site-packages (0.6.3)`

**Reply by deniz7z** [reply]
> @deniz7z are you running Python 3.8? If so, 0.6.x was the last version of PyVRP supporting that Python version. Newer releases of PyVRP target 3.9 and up (3.10 since PyVRP 0.10.0).

Yes I am running 3.8.8. Will update and let you know. Thanks a lot!

**Reply by deniz7z** [reply]
Thanks @leonlan & @N-Wouda.
Seems like the age-old advice to update or restart is valid once again :')

**Reply by N-Wouda** [reply]
Great you got it working!
